java/flight/flight-core/src/test/java/org/apache/arrow/flight/FlightTestUtil.java

```diff
@@ -47,7 +48,11 @@ static Path exampleTlsRootCert() {
 
   static List<CertKeyPair> exampleTlsCerts() {
     final Path root = getFlightTestDataRoot();
-    return Arrays.asList(new CertKeyPair(root.resolve("cert0.pem").toFile(), root.resolve("cert0.pkcs1").toFile()),
+    final Path cert0Pem = root.resolve("cert0.pem");
+    if (!Files.exists(cert0Pem)) {
+      throw new RuntimeException(cert0Pem + " doesn't exist. Take a look on https://arrow.apache.org/docs/dev/developers/java/building.html#building to initialize.");
```

Comment:
```suggestion
      throw new RuntimeException(cert0Pem + " doesn't exist. Make sure submodules are initialized (see https://arrow.apache.org/docs/dev/developers/java/building.html#building)");
```

Reply:
Something like this, in case the user already knows git?

Reply:
Good idea. Let me do the change. Thanks again !

Reply:
Done, thanks again for the suggestion.